.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: csharp
+solution: OdooDotNet.sln
+addons:
+  apt:
+    packages:
+      - expect-dev  # provides unbuffer utility
+      - python-lxml  # because pip installation is slow
+      - python-simplejson
+      - python-serial
+
+env:
+  global:
+  - VERSION="8.0"
```

Comment:
Are you planning to support other versions?
If that's the case, you may want to set up a env matrix for the various versions.
```yml
env:
  matrix:
  - VERSION="6.1"
  - VERSION="7.0"
  - VERSION="8.0"
```